### ISSUE TITLE
Training Resumption

### ISSUE THREAD

**Issue by code4geoai** [opengeos/geoai#207]
I am using this code to run the training process, save the training checkpoints and resume training, but somehow neither the checkpoints are saved now the training is resumed. Am i missing some thing here?
# Train U-Net model
geoai.train_segmentation_model(
    images_dir=f"{out_folder}/images",
    labels_dir=f"{out_folder}/labels",
    output_dir=f"{out_folder}/unet_models",
    architecture="unet",
    encoder_name="resnet34",
    encoder_weights="imagenet",
    num_channels=4,
    num_classes=13,
    batch_size=8,
    num_epochs=50,
    learning_rate=0.001,
    val_split=0.2,
    verbose=True,
    checkpoint_path=f"{out_folder}/unet_models/checkpoint.pth",
    resume_training=True,
    plot_curves=True,
)

**Comment by code4geoai** [comment]
* nor the training is resumed. 
Apologies for typo!

**Comment by giswqs** [comment]
Did you try this notebook example: https://opengeoai.org/examples/load_model_checkpoint

Run it without making any changes to see if it works

**Comment by code4geoai** [comment]
Thank you Professor for extending help as always.!
